### PR TITLE
Fix wiki name directory issue while performing create action

### PR DESCRIPTION
**Pull request by naresh-kumar-babu**
- Replace spaces to underlines.
- Remove accented and non-alphanumeric characters

### CODE REVIEW

File: cmd/create/create.go [hunk 1]
```diff
@@ -40,6 +42,11 @@ func NewCmdCreate() *cobra.Command {
 			if name, canastaInfo, err = prompt.PromptUser(name, yamlPath, rootdbpass, wikidbpass, canastaInfo); err != nil {
 				log.Fatal(err)
 			}
+			
+			// Replace spaces to underlines and remove accented and non-alphanumeric characters
+			name = strings.Replace(name, " ", "_", -1)
```
Comment (jeffw16) [inline cmd/create/create.go]: What does the name refer to? Directory name or the wiki name? It does not make sense to have `$wgSitename` be something like "My_New_Canasta_Wiki".

Comment (naresh-kumar-babu) [reply cmd/create/create.go]: It is the Wiki name.

Comment (jeffw16) [reply cmd/create/create.go]: In that case it would be silly to replace the spaces with underscores.

**Review by jeffw16 — CHANGES_REQUESTED** [review]
I have concerns about replacing the spaces with underscores for $wgSitename

**Review by yaronkoren — APPROVED** [review]

**Comment by yaronkoren** [comment]
@jeffw16 - please re-review! The patch is totally different now.

**Pull request merged**